docs/docs/gatsby-on-windows.md

```diff
@@ -48,19 +44,19 @@ npm cache clean -f
 npm install
 ```
 
-For Visual Studio 2017, download [Visual Studio Community 2017](https://visualstudio.microsoft.com/vs/community/) and install the Desktop development with C++ workflow. You can uncheck everything else.
+Para o Visual Studio 2017, baixe [Visual Studio Community 2017](https://visualstudio.microsoft.com/vs/community/) e instale o desenvolvimento Desktop com fluxo de trabalho C++. Você pode desmarcar todo o resto.
```

Comment:
Só queria comentar que eu não sei se faz sentido traduzir `Desktop development with C++` já que é uma opção pra ser selecionada. Mas como eu não sei a tradução dela, estou indeciso. O que tu acha?

Reply:
ah sim, já que é uma opção, é melhor deixar em itálico pra sinalizar que é um texto retirado de outro lugar.